### ISSUE TITLE
make the welcome message toggleable

### ISSUE THREAD

**Issue by writeescape** [TownyAdvanced/Towny#2921]
theres a message when a new player joins a towny server which i would like to be toggle able

A easy fix would be to check if the language file as a string defined for the message that is longer then 2 characters and if not for it to not bother displaying anything not even a blank line.

**Comment by LlmDl** [comment]
I'll make it a config option. Will be an easy fix.

**Comment by LlmDl** [comment]
Fixed as of 0.92.0.12:

```
    - New Config Option: resident_settings.is_showing_welcome_message
      - default: true
      - If true, players who join the server for the first time will cause the msg_registration message in the language files to be shown server-wide.
```
